src/c-stdaux.h

```diff
@@ -57,7 +57,9 @@ extern "C" {
 #include <limits.h>
 #include <stdalign.h>
 #include <stdarg.h>
+#if !defined(__GNUC__) || __GNUC__ > 4 || __GNUC_MINOR__ > 8
 #include <stdatomic.h>
+#endif
```

Comment:
Does 4.8 define `__STDC_NO_ATOMICS__`? Can we use that instead?

Reply:
according to my tests, it does not. That is also discussed at https://gcc.gnu.org/bugzilla/show_bug.cgi?id=58016